src/backend/input.rs

```diff
@@ -0,0 +1,31 @@
+use std::hash::Hash;
+
+#[derive(Hash, PartialEq, Eq, Clone, Copy)]
+pub enum InputDevice<T: PartialEq + Eq + Hash + Clone + Copy>
```

Comment:
What implications do multiple seats have for the user? I have seen that multiple seats *can* mean multiple cursors and window focuses (https://www.youtube.com/watch?v=WO2L_ihO_rI), but is this required or can a compositor just *merge* all of them into one pointer and input for a given focused window, if it wants to do that.

My primary use case for this is having multiple keyboards attached (laptop and external) with different layouts. And switching layouts is really not much fun with `wlc` currently and I want to have the possibility to configure this transparently for the user, without him dealing with a multi-seat environment.

Also do maybe just want to pass a `WlSeat` object instead of the Id in that case?